app/models/cart.rb

```diff
@@ -10,6 +10,20 @@ def add_item(item_id)
     @data[item_id] += 1
   end
 
+  def update_item_count(session_cart, item_id, params_format)
+    @cart = Cart.new(session_cart)
+    if params_format == "1"
+      @cart.increase_quantity(item_id)
+      session_cart = @cart.data
+    elsif 
+      @cart.decrease_quantity(item_id)
+        if @cart.data[item_id] == 0
+          session_cart.delete(item_id)
+        end
+      session_cart = @cart.data
```

Comment:
Useless assignment to variable - `session_cart`.<br>Inconsistent indentation detected.